PIconnect/PIData.py

```diff
@@ -187,6 +195,31 @@ def current_value(self):
         Return the current value of the attribute."""
         return self._current_value()
 
+    def update_value(
+        self,
+        value,
+        ddatetime: Optional[datetime] = None,
+        update_mode: UpdateMode = UpdateMode.NO_REPLACE,
+        buffer_mode: BufferMode = BufferMode.BUFFER_IF_POSSIBLE,
+    ):
+        """Update value for existing PI object.
+
+        Args:
+            value: value type should be in cohesion with PI object or
+                it will raise PIException: [-10702] STATE Not Found
+            ddatetime (datetime, optional): it is not possible to set future value,
+                it raises PIException: [-11046] Target Date in Future.
+
+        You can combine update_mode and ddatetime to change already stored value.
+        """
+
+        if ddatetime:
+            assert ddatetime.tzinfo, "No timezone specified for the datetime passed."
```

Comment:
Could you please change this to an `if ... raise` construction, raising a ValueError with that message? Codacy warns that `assert` statement can be removed if someone compiles the package to optimised byte code.

Reply:
hm-m, I removed assert. No raise Exception instead. 

With my PI Server, it merely writes a value with datetime of UTC